client/src/components/BubbleTags.tsx

```diff
@@ -1,159 +1,140 @@
-import { useState, useEffect } from 'react';
-import styles from '../components/BubbleTags.module.css';
-import { useMusicContext } from '../context/MusicContext'
-
-
-
-
-
+import { useState, useEffect } from "react";
+import styles from "../components/BubbleTags.module.css";
```

Comment:
your css file is in the same folder than you tsx file, not sure you find it with this path